scripts/cmake/Utils.cmake

```diff
@@ -1,7 +1,6 @@
 # Module to utilities used throughout the project
 
-
-set(ESBMC_BIN "${CMAKE_BINARY_DIR}/src/esbmc/esbmc")
+set(ESBMC_BIN "${CMAKE_INSTALL_PREFIX}/bin/esbmc")
```

Comment:
This breaks my workflow, why is it needed?

Reply:
> This breaks my workflow

Could you please clarify which specific workflow? It is passing on CI, where we have a clean build.

> why is it needed?

astgen.py is installed at CMAKE_INSTALL_PREFIX/resources and referenced using a relative path from the esbmc binary folder in python_language.cpp. However, when using CMAKE_BINARY_DIR, the binary referenced during test execution is in the build folder (where CMake artifacts are generated). Consequently, astgen.py cannot be found, causing failures in the Python tests.

Reply:
> > This breaks my workflow
> 
> Could you please clarify which specific workflow? It is passing on CI, where we have a clean build.

My workflow is: `mkdir build && cd build && cmake [some-weird-flags] .. && ninja && ctest`. In essence, there is no `ninja install`, because up to now it was unnecessary.

> astgen.py is installed at CMAKE_INSTALL_PREFIX/resources

Indeed. In case we make the bundling/extraction work (as per the other comment somewhere below), I think this relative path would go away and `astgen.py` could live somewhere in `/tmp` (or whatever it is called on Windows).

Reply:
> I think this relative path would go away

Done, so I reverted this modification.